webapp/pages/profile/_id/_slug.vue

```diff
@@ -331,6 +331,10 @@ export default {
     },
   },
   methods: {
+    removePostFromList(index) {
+      this.activePosts.splice(index, 1)
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-15T06:49:29Z" title="Monday, July 15th 2019, 8:49:29 am +02:00">Jul 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -331,6 +331,10 @@ export default {
     },
   },
   methods: {
+    removePostFromList(index) {
+      this.activePosts.splice(index, 1)
```


From where you get the `index` parameter, dear? @roschaefer 
After you deleted it from the `for loop` above …

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-15T16:15:54Z" title="Monday, July 15th 2019, 6:15:54 pm +02:00">Jul 15, 2019</time>_

---


~The index parameter is still there! You pass the function `removePostFromList` and it has the signature of expecting one argument. When it's called it gets called with the required index param.~

@Tirokk you are absolutely right! I was assuming that `removePostFromList` was emitting the `index`, but no, that's the task of the page component. Stupid JS didn't crash but removed the first post in the list always.